bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java

```diff
@@ -137,9 +137,12 @@ public void handleWebHookEvent(KonnectedModuleGson event) {
                 // check if the itemType has been defined for the zone received
                 // check the itemType of the Zone, if Contact, send the State if Temp send Temp, etc.
                 if (channelType.contains(CHANNEL_SWITCH) || channelType.contains(CHANNEL_ACTUATOR)) {
-                    OnOffType onOffType = event.getState().equalsIgnoreCase(getOnState(channel)) ? OnOffType.ON
-                            : OnOffType.OFF;
-                    updateState(channelId, onOffType);
+                    logger.debug("The event state is: {}", event.getState());
```

Comment:
A local variable could be introduced for containing event state so that `getState()` wouldn't be called three times. This would also prevent a situation where state could change between calls.

Reply:
Sounds good. The value state can not change, but it's probably a good practice to do it that way.

I also changed the type of onvalue to integer in the config. It was annoying that it was not typed properly. The Konnected panel sends a numeric value.